lib/kbsemi.gi

```diff
@@ -788,7 +783,9 @@ InstallMethod( ShallowCopy,
                   reduced := false,
                   tzrules:= StructuralCopy(kbrws!.tzrules),
                   pairs2check:= [],
-                  ordering :=kbrws!.ordering)));
+                  ordering :=kbrws!.ordering,
+                  freefam := kbrws!.freefam,
+                  tzordering := kbrws!.tzordering)));
```

Comment:
Could you briefly explain or at least hint what issue this fixes (e.g. in the commit message)? Presumably something like "Previously, a shallow copy of a KnuthBendixRewritingSystem was incomplete, and e.g. calling FOOBAR on it resulted in an error"?